webapp/src/index.js

```diff
@@ -66,11 +66,15 @@ export default class Plugin {
             return frontendListName;
         };
 
-        const refresh = ({data: {lists}}) => lists.forEach((listName) => store.dispatch(list(false, getFrontendListName(listName))));
+        const refresh = ({data: {lists}}) => {
+            lists.forEach((listName) => store.dispatch(list(false, getFrontendListName(listName))));
+            store.dispatch(count());
```

Comment:
Ideally we don't have to call this API call here, and instead can provide this data through the websocket event, and not have any API calls here